express/scripts/martech.js

```diff
@@ -373,6 +371,11 @@ async function showRegionPicker() {
   const $div = createTag('div');
   $div.innerHTML = html;
   const $regionNav = $div.querySelector('nav.language-Navigation');
+  if (!$regionNav) {
+    return;
```

Comment:
Shouldn't it "wait" for the the nav to load ?

Reply:
This is not the FEDS nav, but the language `<nav>` element on www.adobe.com. It is already in the DOM there, but CORS will prevent loading it on hlx.page and hlx.live. This check prevents a JS error trying to access an inexistent element.